src/Viz.js

```diff
@@ -132,6 +132,7 @@ export default class Viz extends BaseClass {
     this._legendCutoff = 1;
     this._legendPadding = defaultPadding;
     this._legendPosition = "bottom";
+    this._legendSort = (a, b) => a - b;
```

Comment:
...does this actually work? aren't `a` and `b` JavaScript data objects? Can you try to add a default sorting that uses the `drawLabel` function to sort alphabetically (this is just pseudo-code, I haven't tested this):

```js
this._drawLabel(a).localeCompare(this._drawLabel(b))
```

Reply:
Right, this wasn't working by default. But I wasn't sure which data key to use to compare objects. I tested it by overriding legendSort while creating the plot.

Reply:
`drawLabel` function takes 2 parameters, `d, i`, while we have only `d`. Should I just pass `d` then? That seems wrong though.

Reply:
I think, at least for now, it's okay to not pass `i` (as long as it works with the default settings!)